lib/SPIRV/SPIRVToLLVMDbgTran.cpp

```diff
@@ -758,6 +762,9 @@ DINode *SPIRVToLLVMDbgTran::transImportedEntry(const SPIRVExtInst *DebugInst) {
   DIFile *File = getFile(Ops[SourceIdx]);
   auto *Entity = transDebugInst<DINode>(BM->get<SPIRVExtInst>(Ops[EntityIdx]));
   if (Ops[TagIdx] == SPIRVDebug::ImportedModule) {
+    if (!Entity)
+      return Builder.createImportedModule(Scope, (DIImportedEntity *)nullptr,
+                                          File, Line);
```

Comment:
This change seems to be a duplicate of #951. I suggest to remove this change from this PR, because it is not related to the title of the PR/commit. Note: if we do that, we need to update the test as well.